src/App.js

```diff
@@ -1,25 +1,20 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import BitByteNode from './BitByteNode.jsx'
 
 function App() {
+
+  /* Declaring as a variable for function call */
+  var node1 = <BitByteNode 
+    nodeName = "Waggie Wao"
+    treeName = "Bug - free tree"
+    since = "WI 20"
+    college = "Sixth"
+    major = "Computer Science"
+    class = "2022"
```

Comment:
Gonna reference @shravanhariharan2's PR for his model here: he's still working on it as well but we should ideally match both https://github.com/acmucsd/green-team-backend/pull/1/files

Reply:
Sounds good, I went ahead and matched the props with shravan's model.
I matched the variable names too, except that "name" was a reserved global in react, therefore had to use "node_name" instead

Reply:
Seems good! Next can you add prop-types associated with all the props in the BitByteNode component. Here's an example https://github.com/acmucsd/membership-portal-ui/blob/master/src/components/RegisterForm/index.jsx

Reply:
You'll need to npm install an external package as well since it isn't built into React